app/plugin_api.go

```diff
@@ -933,28 +934,45 @@ func (api *PluginAPI) PluginHTTP(request *http.Request) *http.Response {
 func (api *PluginAPI) CreateCommand(cmd *model.Command) (*model.Command, error) {
 	cmd.CreatorId = ""
 	cmd.PluginId = api.id
+	cmd.Trigger = strings.ToLower(cmd.Trigger)
 
-	cmd, appErr := api.app.CreateCommand(cmd)
+	teamCmds, err := api.app.Srv().Store.Command().GetByTeam(cmd.TeamId)
```

Comment:
I'm worried about the code duplication with `App.CreateCommand` here. Should we factor out the common code instead?

Reply:
Yep, that's better now. Thanks!